layouts/partials/head.html

```diff
@@ -12,7 +12,7 @@
         {{- end }}
     {{- end }}
 
-    <meta name="description" content="{{ .Description | default (.Summary | plainify | htmlUnescape | replaceRE "\\s+" " ") | default .Site.Params.description | truncate 150 }}" />
+    <meta name="description" content="{{- .Description | default (.Summary | plainify | htmlUnescape | replaceRE "\\s+" " " | replaceRE "^ +| +$" "") | default .Site.Params.description | truncate 150 -}}" />
```

Comment:
`replaceRE "^ +| +$" ""` strips leading and trailing spaces.

A subsequent patch will move this all to a different file with proper commenting to explain each step of the transformation.